spras/prm.py

```diff
@@ -1,32 +1,34 @@
-#
-# Implement an abstract PRM class
-#
-#
 from abc import ABC, abstractmethod
 
+from spras.dataset import Dataset
+
 
 class PRM(ABC):
-    def __init__(self, params):
-        self.name = params['name']
-        self.inputdir = params['inputdir']
-        self.outputdir = params['outputdir']
-        self.params = params['params']
-        # debugging
-        #print('Params:',self.params)
```

Comment:
Are you removing these because they weren't used anywhere? Or because they shouldn't be in an abstract class?

Reply:
Both - I also remember that this was actively discouraged from Thursday last week as an OOP-paradigm.